src/medgen2obo.pl

```diff
@@ -104,9 +104,49 @@
 my @ids = keys %th;
 @ids = sort @ids;
 foreach my $id (@ids) {
+    if ($id =~ /^C\d+/) {
```

Comment:
### `UMLS:` and `Medgen:` IDs
We decided at our last meeting that:
- If the ID starts with `C`, create 2 classes for it: 1 with `UMLS:` and 1 with `Medgen:` prefixes.
- Else (if starts with `CN`), just use the `Medgen:` prefix.

The easiest way for me to do this was to edit Chris' Perl. I would have created a function to help wi this, but I've never written Perl before and don't know how to do that, so for nowI copy/pasted in a largely duplicate code block.

Reply:
> If the ID starts with C, create 2 classes for it: 1 with UMLS: and 1 with Medgen: prefixes.

Small correction: UMLS: and MEDGEN_CUI:

> Else (if starts with CN), just use the Medgen: prefix.

CN are _also_ `MEDGEN_CUI`.  use `MEDGEN` only for medgen internal IDS.

Reply:
I've since created a function to do tis in Perl.

Reply:
Closing the loop on this topic. We recently changed namespace capitalization to "MEDGEN" and "MEDGENCUI", and modified which classes went under each namespace. Marking resolved.